tags/resources/tag-formation-templates/template-leadership-election-process.md

```diff
@@ -1,3 +1,15 @@
 # Leadership election process within the `<TAG NAME>`
 
 *tbd* - contributions are welcome!
+
+## Chair
+
+*tbd* - contributions are welcome!
+
+## Technical Lead
+
+*tbd* - contributions are welcome!
```

Comment:
One thought I'd like to share here is that it would be beneficial to have a more data driven/evidential approach to identifying tech lead candidates. The thinking here is that whilst community activities (chop wood and carry water) are valuable, having deep seated expertise is a fantastic way to build trust with projects and act as the bridge between community and contributor.

I feel a fair way to assist in this might be to ask any nominee to contribute three examples of their work within the field that demonstrate aptitude and appropriateness for the role. As part of the election process this also gives the committee more substantive information to elect the appropriate representative.